README.MD

```diff
@@ -0,0 +1,326 @@
+const BrandedToken = require('./index.js');
+const Setup = BrandedToken.EconomySetup;
+
+```
+
+######  Constants
+
+Please can set below constants to addresses that you are able to control.
+
+```js
+
+// Initialize web3 object using the geth endpoint
+const web3Provider = new Web3('http://127.0.0.1:8545');
+
+// organization owner. Doesn't need to be eth funded.
+let organizationOwner = '0xaabb1122....................';
+
+// deployer address
+let deployerAddress = '0xaabb1122....................';
+
+// Facilitator address
+let facilitator = '0xaabb1122....................';
+
+// staker address
+let staker = '0xaabb1122....................';
+
+// worker address
+let worker = '0xaabb1122....................';
+
+let passphrase = 'some passphrase.....';
+
+// Other constants
+const gasPrice = '0x12A05F200';
+const gas = 7500000;
+
+```
+
+###### ABI and BIN provider
+
+brandedtoken.js comes with an in-built abi-bin provider for managing abi(s) and bin (s).
+
+The abiBinProvider provides developers with following abi(s) and bin(s):
+
+* [BrandedToken](https://github.com/OpenSTFoundation/brandedtoken-contracts/blob/0.10.0-alpha.1/contracts/BrandedToken.sol) (BrandedToken contract deployed on ValueChain)
+* [UtilityBrandedToken](https://github.com/OpenSTFoundation/brandedtoken-contracts/blob/0.10.0-alpha.1/contracts/UtilityBrandedToken.sol) (UtilityBrandedToken contract deployed on UtilityChain)
+* [GatewayComposer](https://github.com/OpenSTFoundation/brandedtoken-contracts/blob/0.10.0-alpha.1/contracts/GatewayComposer.sol) (GatewayComposer contract deployed on ValueChain per staker)
+
+```js
+
+// Get the MockToken ABI
+let abiBinProvider = openST.abiBinProvider();
+const mockTokenAbi = abiBinProvider.getABI('BrandedToken');
+
+```
+
+
+#### Deployment of contracts
+To stake BrandedToken using GatewayComposer, you will need an EIP20 contract. You can
+either use a pre-deployed EIP20 contract or deploy a new one as shown below (presently only deploys a new MockToken contract).
+
+##### Deploy EIP20Token contract
+
+The Mock Contract stands in place of the Value Token.If you have a pre-deployed ValueToken, please skip this step.
+
+```js
+const MockContractsDeployer = require('./test/utils/MockContractsDeployer');
+const mockContractDeployerObject = new MockContractsDeployer(deployerAddress, web3Provider);
+let eip20Token;
+mockContractDeployerObject.deployMockToken().then(function(){
+  eip20Token = mockContractDeployerObject.addresses.MockToken;
+});
+```
+
+
+##### Deploy Organization contract
+
+An Organization contract serves as an on-chain access control mechanism by assigning roles to a set of addresses.
+
+```js
+// Deploy Organization contract
+const OrganizationHelper = Setup.OrganizationHelper;
+const organizationHelperObject = new OrganizationHelper(web3Provider);
+const organizationConfig = {
+  deployer: deployerAddress,
+  owner: organizationOwner,
+  workers: worker,
+  workerExpirationHeight: '20000000' // High expiration limit
```

Comment:
"High expiration limit" editorializes and lacks context, so it is not clear to the reader what it means.

Suggest replacing with a comment that simply objectively explains what it is or, at least, the unit of measure.

Reply:
Attempted clarification, please review